src/tokenizer.rs

```diff
@@ -648,6 +649,24 @@ impl<'a> Tokenizer<'a> {
                         return Ok(s);
                     }
                 }
+                '\\' => {
```

Comment:
As postgres's quoting is different (e.g. it uses `''` instead of `\'` I think this case should be wrapped in a `if dialect_of!(self is MySqlDialect)` or something